tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/domain/ExerciseBuilder.java

```diff
@@ -10,17 +10,20 @@
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class ExerciseBuilder {
 
-    private static final String BEGIN_SOLUTION_TAG = "// BEGIN SOLUTION";
-    private static final String END_SOLUTION_TAG = "// END SOLUTION";
-    private static final String STUB_TAG = "// STUB:";
-    private static final String SOLUTION_FILE_TAG = "// SOLUTION FILE";
+    private static final String BEGIN_SOLUTION_REGEX = ".*\\/\\/[ \\t]?BEGIN[ \\t]SOLUTION.*";
```

Comment:
The ?'s were relics from my first version of the regex. Changed them and added + to the spaces between words, to make sure there always is a space but not take stand on how many.